dist/contact/index.html

```diff
@@ -73,12 +73,12 @@
           <div>
             <h2>David Ford</h2>
             <p>Working in UK Pensions since 1980, David emigrated from the UK to Australia in 1996 and is a leading expert in Pension Transfers to Australia.</p>
-            <a class="button" href="tel:+61424777888" title="Call David Ford"><span class="name">Call David</span><span class="phone">0424 777 888</span></a>
+            <a class="button" href="tel:0428 041 896" title="Call David Ford"><span class="name">Call David</span><span class="phone">0428 041 896</span></a>
```

Comment:
In the `tel:...` always use international number with no spaces, no (), because several browsers break if you don't. So use `tel:+61428041896` instead. That number is the one that will popup in your phone when clicking the link.